pkg/router/controller/host_admitter.go

```diff
@@ -143,7 +143,6 @@ func (p *HostAdmitter) HandleRoute(eventType watch.EventType, route *routev1.Rou
 		switch eventType {
 		case watch.Added, watch.Modified:
 			if err := p.addRoute(route); err != nil {
-				log.Error(err, "route not admitted", "routeNameKey", routeNameKey(route))
```

Comment:
Was this change accidentally committed, or is this being removed for a specific reason?

Reply:
This is being removed because it is not needed, the err is returned.  I checked with Miciah about removing it due to its noisiness in the log, and he agreed.